package/gluon-radv-filterd/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-radv-filterd
+PKG_VERSION:=1
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(GLUONDIR)/include/package.mk
```

Comment:
For LEDE this needs to be changed to `include ../gluon.mk` (2dd8a700ca9cc288dc4ea8abb261b8e210c446f6).